src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/hosts/hosts.component.ts

```diff
@@ -198,6 +198,18 @@ export class HostsComponent extends ListWithDetails implements OnDestroy, OnInit
         flexGrow: 1,
         cellTemplate: this.hostNameTpl
       },
+      {
+        name: $localize`Status`,
+        prop: 'status',
+        flexGrow: 0.8,
+        cellTransformation: CellTemplate.badge,
+        customTemplateConfig: {
+          map: {
+            maintenance: { class: 'badge-warning' },
+            available: { class: 'badge-success' }
+          }
+        }
+      },
```

Comment:
status column is something that hasn't provide much value to the user apart from showing the `maintenance` and `offline` status which are the only real status we get. Whereas `Service Instances` gives a lot more value? so why prioritize the status?

Reply:
maybe we don't even need a `status` column. We could just represent them with 3 icons.

:heavy_check_mark: when its available (or no status is being provided from cephadm)
:information_source: when its in maintenance
:x: when the host is offline

we could map these as success, warning and danger icons and put them right next to the name (same as the subvolume listing)

Reply:
IMO: The status of something always has the priority as it represents if something is working or not and that is first thing you want to check before looking for anything. 

Also have a separate column to represent it would be good as this is first thing a user wants to see if something goes wrong in cluster and see if the Node is up or not.

Reply:
in most cases I'd agree. But here the status wouldn't be the same as that. Because the only real error we are going to get is `offline` and the maintenance is what user himself adds on purpose. As a user, I'd prefer to have priority to Service Instances because it gives some valuable information. If cephadm gives us the lifecycle status of the host at some point, maybe we could use it. Because currently we are displaying `available` when the host is not even ready (meaning there are still actions ongoing inside the host)